tests/Feature/Concerns/HandlesAiFeatureResponsesTest.php

```
@@ -73,6 +74,27 @@
+
+    expect( $outcome->succeeded )->toBeFalse()
+        ->and( $outcome->feature )->toBe( 'ai.alt_text' )
+        ->and( $outcome->output )->toBeNull()
+        ->and( $outcome->status )->toBe( 429 )
+        ->and( $outcome->errorCode )->toBe( 'budget_exceeded' )
+        ->and( $outcome->statusSlug )->toBe( 'budget-exceeded' )
+        ->and( $outcome->message )->toBe( 'The AI monthly budget has been reached. Please try again later.' )
+        ->and( $outcome->message )->not->toContain( '120' )
+        ->and( $outcome->message )->not->toContain( '100' );
+
+    Log::shouldNotHaveReceived( 'error' );
```

PHPStan (2.2.8)

[error] Call to an undefined static method Illuminate\Support\Facades\Log::shouldNotHaveReceived().


(staticMethod.notFound)